enable/tools/drag_tool.py

```diff
@@ -24,10 +24,11 @@ class DragTool(BaseTool):
     # The mouse button used for this drag operation.
     drag_button = Enum("left", "right")
 
-    # End the drag operation if the mouse leaves the associated component?
+    # Do nothing, cancel or end the drag operation if the mouse leaves the
+    # associated component?
     # NOTE: This behavior depends on "mouse_leave" events, which in general
     # are not fired when `capture_mouse` is True (default).
```

Comment:
I dont think this note is correct?  If I leave `capture_mouse` as true, but set `end_drag_on_leave` to true and mess around with examples, I am seeing mouse_leave events fired and the drags cancelled on leave?
Maybe I am missing something

Reply:
It might be different between Qt and Wx.

Reply:
Completely orthogonal, but this just made me remember that chaco no longer lists wx as being supported in its `ci/edmtool.py` / wx is not exercised on its CI ...